packages/opencensus-instrumentation-http/src/http-metrics.ts

```diff
@@ -0,0 +1,239 @@
+    'Server-side total bytes received in request body (uncompressed)');
+
+/**
+ * {@link Measure} for the server-side total bytes sent in response bodies
+ * (not including headers but including error responses with bodies). Should
+ * be measured from actual bytes written and sent, not the value of the
+ * Content-Length header. This is uncompressed bytes. Responses with no
+ * body should record 0 for this value.
+ */
+export const HTTP_SERVER_SENT_BYTES = globalStats.createMeasureInt64(
+    'opencensus.io/http/server/sent_bytes', MeasureUnit.BYTE,
+    'Server-side total bytes sent in response bodies (uncompressed)');
+
+/**
+ * {@link Measure} for the server-side time between first byte of request
+ * headers received to last byte of response sent, or terminal error.
+ */
+export const HTTP_SERVER_LATENCY = globalStats.createMeasureDouble(
+    'opencensus.io/http/server/server_latency', MeasureUnit.MS,
+    'Server-side time between first byte of request headers received to last byte of response sent, or terminal error');
+
+/** {@link TagKey} for the value of the client-side HTTP host header. */
+export const HTTP_CLIENT_HOST = {
+  name: 'http_client_host'
+};
+
+/** {@link TagKey} for the value of the server-side HTTP host header. */
+export const HTTP_SERVER_HOST = {
+  name: 'http_server_host'
+};
+
+/**
+ * {@link TagKey} for the numeric client-side HTTP response status code (e.g.
+ * 200, 404, 500). If a transport error occurred and no status code was read,
+ * use "error" as the {@code TagValue}.
+ */
+export const HTTP_CLIENT_STATUS = {
+  name: 'http_client_status'
+};
+
+/**
+ * {@link TagKey} for the numeric server-side HTTP response status code (e.g.
+ * 200, 404, 500). If a transport error occurred and no status code was written
+ * use "error" as the {@code TagValue}.
+ */
+export const HTTP_SERVER_STATUS = {
+  name: 'http_server_status'
+};
+
+/**
+ * {@link TagKey} for the client-side URL path (not including query string) in
+ * the request.
+ */
+export const HTTP_CLIENT_PATH = {
+  name: 'http_client_path'
+};
+
+/**
+ * {@link TagKey} for the server-side URL path (not including query string) in
+ * the request.
+ */
+export const HTTP_SERVER_PATH = {
+  name: 'http_server_path'
+};
+
+/**
+ * {@link TagKey} for the client-side HTTP method of the request, capitalized
+ * (GET, POST, etc.).
+ */
+export const HTTP_CLIENT_METHOD = {
+  name: 'http_client_method'
+};
+
+/**
+ * {@link TagKey} for the server-side HTTP method of the request, capitalized
+ * (GET, POST, etc.).
+ */
+export const HTTP_SERVER_METHOD = {
+  name: 'http_server_method'
+};
+
+/**
+ * {@link TagKey} for the server-side logical route, a pattern that matched the
+ * URL, of a handler that processed the request.
+ */
+export const HTTP_SERVER_ROUTE = {
+  name: 'http_server_route'
+};
+
+const SIZE_DISTRIBUTION: number[] = [
+  0, 1024, 2048, 4096, 16384, 65536, 262144, 1048576, 4194304, 16777216,
```

Comment:
Not sure if we still want to include the leading 0s?

Reply:
I would keep it now, until we officially remove this from the specs. 
Context (for the record) -> we are already [dropping negative](https://github.com/census-instrumentation/opencensus-node/blob/2df1bc4293dbfe43081bed8e98cea551f6602f5c/packages/opencensus-core/src/stats/bucket-boundaries.ts#L65) and zero bucket bounds in top level API, so it doesn't make sense to include 0 at first place. Also [specs](https://github.com/census-instrumentation/opencensus-proto/blob/master/src/opencensus/proto/metrics/v1/metrics.proto#L223) says "**The values must be strictly increasing and > 0**".

Reply:
Lets remove it. We can open another PR to update the specs. For now we have two conflicting specs (one for metrics proto and one for http/grpc stats). Since we don't have any useful data in zero bucket might as well remove it.

Reply:
Ok, done.